### DISCUSSION TITLE
Please reinforce your documentation with better examples

### DISCUSSION THREAD

**Discussion by JTalarico** [Akryum/v-tooltip#994] [Ideas]
The documentation is seriously lacking. I have never had such a difficult time getting such a simple component up and running.

The styling is extremely unclear.

**Comment by chs-zalam** [comment]
+1

**Comment by Akryum** [comment]
Which parts of the docs are confusing to you? Did you have trouble with the [Getting started](https://floating-vue.starpad.dev/guide/)?

**Reply by chs-zalam** [reply]
I had problems with applying custom styles which ultimately got resolved using css nesting. 

Anyway, appreciate your contribution to Vue community.

**Reply by jlecointe** [reply]
Where is the documentation for VMenu/Menu/SubMenu component please?
